core/src/main/scala/org/apache/spark/scheduler/SparkListener.scala

```diff
@@ -98,6 +99,9 @@ case class SparkListenerExecutorAdded(time: Long, executorId: String, executorIn
 case class SparkListenerExecutorRemoved(time: Long, executorId: String, reason: String)
   extends SparkListenerEvent
 
+@DeveloperApi
+case class SparkListenerBlockUpdated(updateBlockInfo: UpdateBlockInfo) extends SparkListenerEvent
```

Comment:
Isnt UpdateBlockInfo private[spark]? Then its not right that a public class refer to an internal class that people will not be able to refer to.